orm_lib/src/TransactionImpl.cc

```diff
@@ -41,25 +41,26 @@ TransactionImpl::~TransactionImpl()
         auto loop = connectionPtr_->loop();
         loop->queueInLoop([conn = connectionPtr_,
                            ucb = std::move(usedUpCallback_),
-                           commitCb = std::move(commitCallback_)]() {
+                           commitCb = std::move(commitCallback_),
+                           autoCommit = autoCommit_]() {
```

Comment:
UB if the user manually commits then auto commit happens. Could the following happen?

```c++
auto t = db->newTransactionCoro();
...
t->commit();
// now `t` destructs. Calls auto commit. We try to move an already moved variable.
```